environment.yml

```diff
@@ -1,28 +1,37 @@
-# Install the course conda environment with the following command:
-# conda env create -f env-dsci-571.yml
-# Once you install it, you should be able to see it in your Jupyter Lab.
-# If not, check the documentation here: https://ipython.readthedocs.io/en/stable/install/kernel_install.html
-name: "522"
-
+name: '522'
 channels:
   - conda-forge
   - pytorch
   - anaconda
   - plotly
   - defaults
 dependencies:
-  - python==3.12
-  - ipykernel
-  - matplotlib
-  - pandas
-  - nb_conda_kernels
-  - scikit-learn>=1.5.1
-  - requests
-  - ipython
+  - python=3.12.0
+  - pandas=2.2.3
+  - numpy=1.26.4
+  - scikit-learn=1.5.2
+  - altair=5.4.1
+  - jupyterlab=4.2.6
+  - jupyterlab_server=2.27.3
+  - jupyter_server=2.14.2
+  - jupyter_core=5.7.2
+  - jupyter_client=8.6.3
+  - ipykernel=6.29.5
+  - nb_conda_kernels=2.5.1
+  - matplotlib=3.9.2
+  - scipy=1.14.1
+  - vega_datasets
```

Comment:
Remove `vega_datasets` since we do not need datasets from this package.

Reply:
ok